docs/modules/servers/pages/distributed/concepts.adoc

```diff
@@ -0,0 +1,20 @@
+= James Distributed Server &mdash; Background Concepts
+:navtitle: Concepts
+
+This section describes the concepts upon which the
+**James Distributed Server** was based.
+
+[TODO: provide links to each of these topics - Ensure there are no duplicates etc.]
+
+ * xref:servers/extendable/concepts.adoc[All of the concepts for the Extendable Server]
+ * https://en.wikipedia.org/wiki/NoSQL[NoSQL data storage]
+  ** https://cassandra.apache.org/[Apache Cassandra]
+ * Sharding
+ * Load balancing
+ * Mail Processing
+ * Queuing
+ * Storage
+ * BlobStore
+ * Task Management
+ * Event Processing
```

Comment:
Agree!

Would you agree to the principle that:

 * Extendable Server builds on Basic Server, and
 * Distributed Server builds on Extendable Server?

If so, then we can simply either put in a "see blah" reference, or we could structure the Antora docs to include the relevant content so that it is only written once.

Do you have a preference?

Reply:
> or we could structure the Antora docs to include the relevant content so that it is only written once.

I prefer that one, and I 100% agree on the "only written once"